pkg/runner/runner.go

```diff
@@ -66,32 +68,38 @@ func (e *ErrActionUnknown) Error() string {
 // handles the logic of the utility
 func (r *Runner) Exec(options *flags.Options) (err error) {
 	if options.Action == "create" {
-		log.Infof("Checking current versions for release %s, channel %s and arch %s", options.Release, options.Channel, options.Arch)
-		var siVersion, cloudVersion int
-		siVersion, cloudVersion, err = r.getVersions(options.Release, options.Channel, options.Arch)
-		if err != nil {
-			return
-		}
-		if siVersion <= cloudVersion {
-			return &ErrVersion{siVersion, cloudVersion}
-		}
-		var path string
-		log.Infof("Creating image file in %s", path)
-		path, err = r.imgDriver.Create(options.Release, options.Channel, options.Arch, siVersion)
-		defer os.Remove(path)
-		if err != nil {
-			return
-		}
-		log.Infof("Uploading %s", path)
-		err = r.imgDataTarget.Create(path, options.Release, options.Channel, options.Arch, siVersion)
-		if err != nil {
-			return
-		}
-		log.Infof("Finished", path)
-	} else {
-		return &ErrActionUnknown{action: options.Action}
+		return r.createActions(options)
+	} else if options.Action == "cleanup" {
+		return r.cleanupActions(options)
+	}
+	return &ErrActionUnknown{action: options.Action}
+}
+
+func (r *Runner) createActions(options *flags.Options) (err error) {
```

Comment:
All this code is what gets executed with the "create" action. I think that calling it create is ok, the same for cleanup.